docs/acn-internals.md

```diff
@@ -4,7 +4,7 @@ the main implementation of the Agent Communication Network (ACN).
 
 In particular:
 
-- The <a href="https://github.com/valory-xyz/open-acn" target="_blank">`open-acn`<a/> Golang library; At the moment a copy resides locally in open-aea here as `libp2p_node`. 
```

Comment:
Ignore changes in this file: most of them are blankspaces.  Also there are two link fixes.